src/main/java/team/gif/robot/Robot.java

```diff
@@ -104,32 +114,39 @@ public void disabledPeriodic() {}
     /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
     @Override
     public void autonomousInit() {
-        if (isSwervePBot) {
-            m_autonomousCommand = m_robotContainer.getAutonomousCommand(autoMode.SWERVE_POC);
-        } else {
-            m_autonomousCommand = m_robotContainer.getAutonomousCommand(null);
-        }
-//        m_autonomousCommand = m_robotContainer.getAutonomousCommand(chosenAuto);
+        elapsed.reset();
+        elapsed.start();
+        runAutoScheduler = true;
 
+        autonomousCommand = robotContainer.getAutonomousCommand(chosenAuto);
 
-        // schedule the autonomous command (example)
-        if (m_autonomousCommand != null) {
-            m_autonomousCommand.schedule();
-        }
     }
 
     /** This function is called periodically during autonomous. */
     @Override
-    public void autonomousPeriodic() {}
+    public void autonomousPeriodic() {
+        if (runAutoScheduler && (elapsed.get() > (chosenDelay.getValue()))) {
+            if (isSwervePBot) {
```

Comment:
Do we still need this or can you get the auto from the dashboard?

Reply:
That is something I forgot I could do - will implement

Reply:
remove the if section since it is set in autonomousInit